modules/auxiliary/admin/http/typo3_sa_2010_020.rb

```diff
@@ -3,139 +3,130 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
-require 'thread'
```

Comment:
Just making sure this is suppose to have been removed?

Reply:
yeah, I believe it's been identified as a dead include

```
modules/auxiliary/admin/http/typo3_sa_2010_020.rb:6:1: W: [Corrected] Lint/RedundantRequireStatement: Remove unnecessary require statement.
require 'thread'
^^^^^^^^^^^^^^^^

```

Examples:

https://docs.rubocop.org/rubocop/cops_lint.html#lintredundantrequirestatement

Reply:
Everything looks good, happy to merge 👍